src/contexts/session.context.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, PropsWithChildren, useContext, useEffect, useState } from 'react';
+import { useStore } from '../hooks/store.hook';
+
+interface SessionInterface {
+  authenticationToken?: string;
+  setSession(authenticationToken: string): void;
```

Comment:
I want to add user information in a sort of session object. right now I just have token. so `setSession` doesn't really make sense. yes, but I would keep it, as it will change anyway